text/1017-triggers.rst

```diff
@@ -0,0 +1,174 @@
+::
+
+    Status: Draft
+    Type: Feature
+    Created: 2022-12-08
+    Authors: Michel J. Sullivan <sully@msully.net>
+
+==================
+RFC 1017: Triggers
+==================
+
+This RFC proposes a mechanism for triggers.
+
+Some text stolen from an earlier RFC by Elvis.
+
+Motivation
+==========
+
+Requirements
+============
+
+Specification
+=============
+
+Synopsis::
+
+    type <type-name> "{"
+        trigger <name>
+            on { insert | update | delete } [, ... ]
+            <action>
+    "}"
+
+    # where <action> can be one of:
+
+    DO WHEN <element-expr>
+
+    DO AFTER <set-expr>
+
+    DO ON COMMIT <set-expr>
```

Comment:
I think this should be `DO AFTER COMMIT` to highlight that things happen _after_ commit.